R/data.R

```diff
@@ -162,19 +162,22 @@ process_fitted_params <- function(fitted_params) {
         paths <- file.path(tempdir(), paths)
         chain <- 1
         for (path in paths) {
-          chain_draws <- posterior::as_draws_df(posterior::subset_draws(draws, chain = chain))
-          colnames(chain_draws) <- unrepair_variable_names(variables)
+          chain_draws <- posterior::subset_draws(draws, chain = chain)
+          unname(chain_draws)         
```

Comment:
Two things: 

1. Why do we need to remove the names here? That's fine if necessary, just curious. 

2. Right now this isn't assigned to anything. I think you need 

```
chain_draws <- unname(chain_draws)
```

Reply:
> Why do we need to remove the names here? That's fine if necessary, just curious.

Because it otherwise writes the iteration ids in the CSV.

>Right now this isn't assigned to anything. I think you need

Hm, this did help with that, will double-check.